ops/terraform/env/prod/stateful/main.tf

```diff
@@ -16,6 +16,13 @@ module "stateful" {
     allocated_storage = 16000
   }
 
+  db_params = [
+    {name="max_worker_processes", value="16", apply_on_reboot=true},
+    {name="random_page_cost", value="1", apply_on_reboot=false},
```

Comment:
I think we should also set this related PGTune recommendation for SAN disks:

```
effective_io_concurrency = 300
```

But if you want to do that as part of a separate PR, so we can benchmark it, that's cool.

Reply:
I keep waffling on this one due to a huge lack of publicly posted data but I found a bit more that indicates this should be ok on IO1 type storage, though can be less performant on the less reliable GP2 storage.